src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -979,7 +979,7 @@ internal override IEnumerable<MethodSymbol> GetMethodsToEmit()
                     var method = (MethodSymbol)members[index];
 
                     // Don't emit the default value type constructor - the runtime handles that
-                    if (!method.IsDefaultValueTypeConstructor())
+                    if (!method.IsDefaultValueTypeConstructor(requireZeroInit: false))
```

Comment:
This seems wrong. Feels like we should only skip the zero init constructor, but not any other default value type constructor.

Reply:
Any thoughts?

Reply:
The value of `requireZeroInit:` is ignored for methods from metadata. `IsDefaultValueTypeConstructor()` simply checks if the constructor is implicitly declared. I've added a comment and a test.